go.mod

```diff
@@ -38,11 +38,14 @@ require (
 	github.com/prometheus/client_model v0.2.0 // indirect
 	github.com/prometheus/common v0.10.0 // indirect
 	github.com/prometheus/procfs v0.2.0 // indirect
+	go.uber.org/goleak v1.1.10 // indirect
```

Comment:
Just curious, go mod tidy adds this here? I'm not sure why, but I guess it is ok... Or did you add it manually?